source/features/add-tag-to-commits.tsx

```diff
@@ -134,9 +134,6 @@ async function init(): Promise<void | false> {
 			// No tags for this commit found in the cache, check in github
 			cached = mergeTags(cached, await getTags(lastCommitOnPage)); // eslint-disable-line no-await-in-loop
 			targetTags = cached[targetCommit];
-		}
-
-		if (!targetTags) {
```

Comment:
Please Double check that my change is correct and for some reason it was designed like this.

Reply:
No, this should be undone, as you can see it tries to set targetTags in 2 ways, so it has to check twice.